Xamarin.Forms.Platform.WinRT/ButtonRenderer.cs

```diff
@@ -44,12 +44,10 @@ protected override void OnElementChanged(ElementChangedEventArgs<Button> e)
 				if (Element.BorderColor != Color.Default)
 					UpdateBorderColor();
 
-				if (Element.BorderWidth != 0)
-					UpdateBorderWidth();
-
 				if (Element.BorderRadius != (int)Button.BorderRadiusProperty.DefaultValue)
 					UpdateBorderRadius();
 
+				UpdateBorderWidth();
```

Comment:
Should we check to see if the `Element.BorderWidth != DefaultBorderWidth` here before we try to set it?